EnglishFilter/sections/foreign.txt

```diff
@@ -4395,6 +4395,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/93133
+chosun.com#?#div.raw-html > div iframe[src*="//adex.ednplus.com/"]:upward(5)
```

Comment:
Why not `biz.chosun.com###adbay_floatingid`?
I also added rules that blocks leftovers (desktop / mobile) https://github.com/AdguardTeam/AdguardFilters/commit/756db1553b582d3db9ff661e084165f26cea497c

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/93141/commits/e9c954bae4281744203c6c21cab6107a9a926522

Reply:
Ok, but I already added those rules directly.